hdl/ip/vhd/spi_nor_controller/spi_nor_regs.rdl

```diff
@@ -124,7 +124,7 @@ addrmap nor_flash_regs {
         name = "SP5 Flash offset address";
         desc = "Allows hubris to control which flash page is accessed by the eSPI block";
         field {
-            desc = "Address added to raw eSPI transactions to generate physical flash address";
+            desc = "*Signed* Address added (subtracted for -ve) to raw eSPI transactions to generate physical flash address";
```

Comment:
Maybe just write the the word "negative" here instead of "-ve"? I don't fine -ve to be very intuitive to resolve to negative.

Reply:
Fixed